### ISSUE TITLE
Product detail layout is misaligned

### ISSUE THREAD

**Issue by bigfatbird** [elementary/store#156]
## What Happened

Product layout is not ideal.

## Expected Behavior

Show the product photo aligned with the product-variant option.

## Steps to Reproduce

1. Visit https://store.elementary.io/products/179514140
2. See layout misaligned
3.
<img width="1437" alt="Screenshot 2021-08-31 at 10 47 55" src="https://user-images.githubusercontent.com/1159341/131472723-68540768-3d9a-4d42-b758-889f1f1f05a1.png">

## Platform Information

Firefox 91.0.2 on 100% zoom
macOS 11.5.2
MacBook Pro 15" (2018)

**Comment by bigfatbird** [comment]
Maybe it would be good to implement a dropdown if there are more than 3 or something color options?

**Comment by cassidyjames** [comment]
I think I want to move the colors to just being circles with tooltips for the precise color names—they're primarily visual, as clicking them shoes you an updated thumbnail, so the color name is not critical information.

**Comment by bigfatbird** [comment]
Yes, i guess they are not as important. If you feel the need to inform users about the color name you can do while hovering, or maybe make an animation expanding the color of the selected value.

also they could be printed in the receipt of the order.
